config/resources/domain.json

```diff
@@ -245,44 +234,6 @@
       "features": ["include-uri"],
       "new-resource-base": "http://data.lblod.info/mappings/"
     },
-    "relations": {
-      "name": "relation",
-      "class": "ext:Relation",
-      "attributes": {
-        "expectedNumber": {
-          "type": "integer",
-          "predicate": "ext:expectedNumber"
-        },
-        "reason": {
-          "type": "string",
-          "predicate": "ext:reason"
-        },
-        "order": {
-          "type": "integer",
-          "predicate": "qb:order"
-        }
-      },
-      "relationships": {
-        "concept": {
-          "predicate": "ext:concept",
-          "target": "concept",
-          "cardinality": "one"
-        }
-      },
-      "new-resource-base": "http://data.lblod.info/relations/"
-    },
-    "can-be-combined-with-relations": {
-      "name": "canBeCombinedWithRelation",
-      "class": "ext:CanBeCombinedWithRelation",
-      "super": ["relation"],
-      "new-resource-base": "http://data.lblod.info/can-be-combined-with-relations/"
-    },
-    "must-use-relations": {
-      "name": "mustUseRelation",
-      "class": "ext:MustUseRelation",
-      "super": ["relation"],
-      "new-resource-base": "http://data.lblod.info/must-use-relations/"
-    },
```

Comment:
**Question:** I still found entries for these type in the triplestore. Do we need to delete them via migrations? 

```
SELECT DISTINCT ?type (COUNT(?s) as ?typeCount) WHERE {
  ?s a ?type .

  VALUES ?type {
    <http://mu.semte.ch/vocabularies/ext/Relation>
    <http://mu.semte.ch/vocabularies/ext/CanBeCombinedWithRelation>
    <http://mu.semte.ch/vocabularies/ext/MustUseRelation>
  }
} LIMIT 100
```

Result on my localhost: 
| type | typeCount |
| ----- | --------------- | 
| `http://mu.semte.ch/vocabularies/ext/MustUseRelation` | 8011 |
| `http://mu.semte.ch/vocabularies/ext/Relation` | 3598 |

Reply:
The follow-up ticket: https://binnenland.atlassian.net/browse/GN-4948